package.json

```diff
@@ -27,11 +27,11 @@
     "astro": "^2.10.14",
     "c8": "^8.0.1",
     "clean-publish": "^4.2.0",
-    "eslint": "^8.46.0",
+    "eslint": "^8.48.0",
     "eslint-config-standard": "^17.1.0",
-    "eslint-plugin-import": "^2.28.0",
-    "eslint-plugin-n": "^16.0.1",
-    "eslint-plugin-perfectionist": "^1.5.1",
+    "eslint-plugin-import": "^2.28.1",
+    "eslint-plugin-n": "^16.0.2",
+    "eslint-plugin-perfectionist": "^2.0.0",
```

Comment:
We may need also install explicitly `eslint-plugin-node-import`

Reply:
Done.